invader_payment_pagseguro/services/payment_pagseguro.py

```diff
@@ -241,6 +242,7 @@ def get_schema_confirm_payment_credit(self):
                     "required": True,
                     "schema": {
                         "name": {"type": "string", "required": True},
+                        "brand": {"type": "string", "required": True},
```

Comment:
Deixa a brand como not required, pra manter a compatibilidade com o front atual.

Reply:
feito